### DISCUSSION TITLE
Opti solver options

### DISCUSSION THREAD

**Discussion by riccardocsc** [casadi/casadi#3739] [Q&A] [answered]
Hello everyone!

I'm solving a MPC problem using casadi Opti. Up to now I am able to solve it using ipopt as solver with the call opti.solver('ipopt').
I was wondering if there is a way to change the solver to another one. For example, I know my problem is a QP so I might just go for a QP solver.

Any help is appreciated!

**Comment by sandeep026** [comment, marked as answer]
Use conic.
```
opti=casadi.Opti('conic');
x=opti.variable(2);

opti.subject_to([1 2]*x<=-2);
opti.minimize(sumsqr(x));
opti.solver('qpoases');
sol=opti.solve();

sol.value(x)
```
